site/ja/docs/privacy-sandbox/proposal-lifecycle/index.md

```diff
@@ -0,0 +1,165 @@
+---
+layout: layouts/doc-post.njk
+title: プライバシー サンドボックスの提案のライフサイクル
+date: 2022-03-30
+authors:
+  - alexandrawhite
+---
+
+[提案のライフサイクル](/docs/privacy-sandbox/proposal-lifecycle/)
+
+GitHub English source:
+[https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md](https://raw.githubusercontent.com/GoogleChrome/developer.chrome.com/main/site/en/docs/privacy-sandbox/proposal-lifecycle/index.md)
+
+Google
+がどのようなプロセスで関係者と協力して、プライバシー保護技術についてディスカッション、テスト、導入するかについて。
+
+_こちらのコンテンツの多くは、元々は [2021 年の Chrome Developer Summit のまとめ](/docs/privacy-sandbox/cds21-update/)の一部として共有されていたものです。_
+
+## [プライバシー サンドボックスの提案の目的](/docs/privacy-sandbox/proposal-lifecycle/#what-is-the-goal-of-the-privacy-sandbox-proposals)
```

Comment:
This link is not necessary. Please try to replicate the English version as much as possible. Other headers as well.

Reply:
This comment still stands. Please go over all headings, comparing with the English version and remove links where necessary.

Reply:
What I meant is, to remove the link not to remove the heading itself.